lib/debugbar/engine.rb

```diff
@@ -84,6 +84,27 @@ def log(msg)
       Debugbar::RequestBuffer.init(adapter)
     end
 
+    initializer "debugbar.append_routes" do |app|
+      # Before 0.5.0, I asked people to mount the engine manually
+      # This will probably throw an error at some point to be ultimately removed
+      app.routes.append do
```

Comment:
Notice that we `append` to be able to check of the route was already defined.
You'd get an error if you try to mount the same engine twice.